src/main/webapp/app/app.component.ts

```diff
@@ -17,13 +19,14 @@ export class AppComponent implements OnInit {
 
   tasks$: Observable<Task[]>;
 
-  constructor(@Inject('TaskService') private taskService: TaskService) { }
+  constructor(@Inject('TaskService') private taskService: TaskService, private userService: UserService){ }
 
   ngOnInit(): void {
     this.now$ = timer((60 - new Date().getSeconds()) * 1000, 60 * 1000)
       .pipe(startWith(0))
       .pipe(map(() => new Date()));
     this.tasks$ = this.taskService.getAll();
+    this.userService.get('joseym90@gmail.com').subscribe(user => localStorage.setItem(LocalTaskService.USER_STORAGE_KEY, user.id));
```

Comment:
I guess that should be more generic.

Reply:
Yes it would be more generic when the user login issue have been implemented. Wasn't my issue at the moment.